### PR TITLE
implementing tests for previewEditor and  notelink  component

### PR DESCRIPTION
**Pull request by joshwambere**
## Description

Create tests fro preview editor component
Create tests for notelink component

### Browser checklist

This PR has been tested in the following browsers:

- [X] Chrome
- [ ] Firefox
- [ ] Safari

### Testing checklist

- [X] End-to-end tests have been created if necessary

### CODE REVIEW

File: tests/unit/client/components/editor/EditorEmpty.test.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  it('renders the EmptyEditor component', () => {
+    const component = render(<EmptyEditor />)
+
+    expect(component).toBeTruthy()
+  })
+
+  it('renders the EmptyEditor component and its texts', () => {
+    const component = render(<EmptyEditor />)
+
+    const createNotText = component.queryByTestId(TestID.EMPTY_EDITOR)
```
Comment (taniarascia) [inline tests/unit/client/components/editor/EditorEmpty.test.tsx:19]: NoteText not NotText?

Comment (joshwambere) [reply tests/unit/client/components/editor/EditorEmpty.test.tsx]: issue resolved

Comment (taniarascia) [reply tests/unit/client/components/editor/EditorEmpty.test.tsx]: Thanks!

**Comment by taniarascia** [comment]
@all-contributors add @joshwambere for test

**Pull request merged**